src/audio/host-zephyr.c

```diff
@@ -715,9 +715,7 @@ static int host_verify_params(struct comp_dev *dev,
 }
 
 int host_zephyr_params(struct host_data *hd, struct comp_dev *dev,
-		       struct sof_ipc_stream_params *params, struct list_item *sink_list,
-		       struct list_item *source_list, struct pipeline *pipeline,
-		       uint32_t frames, bool is_scheduling_source)
+		       struct sof_ipc_stream_params *params, bool is_scheduling_source)
```

Comment:
why do you need is_scheduling_source as a param? it can be derived here too

Reply:
and please squash this patch with the previous one

Reply:
ok，will do